### DISCUSSION TITLE
Bug? Error with CloudFormation stack sb-dev; Bucket cannot have ACLs set with ObjectOwnership

### DISCUSSION THREAD

**Discussion by bricewb** [awslabs/aws-saas-boost#509] [Q&A] [answered]
Hello,

I'm suspicious this is a bug. Performed a new install into a fresh account twice and ended up with the same error.

Error: 
Installation Error: Error with CloudFormation stack sb-dev. Check the events in the AWS CloudFormation Console
Please see detailed log file saas-boost-install.log

Looking in the cloud formation event log I see the following on the Logging logical ID.
Bucket cannot have ACLs set with ObjectOwnership's BucketOwnerEnforced setting (Service: Amazon S3; Status Code: 400; Error Code: InvalidBucketAclWithObjectOwnership.

Reason why I think this may be a bug: 
April 2023 S3 started defaulting all new buckets to 'Bucket owner enforced' ownership setting (https://aws.amazon.com/about-aws/whats-new/2022/12/amazon-s3-automatically-enable-block-public-access-disable-access-control-lists-buckets-april-2023/) 

Was really hoping to use this to help develop a new SaaS platform for our company. Hoping to see if anyone else has had this issue and if there are any current work arounds or if a new release will be required. 

Thanks!

**Comment by mariandumitrascu** [comment]
This is caused by some kind of global change in AWS policies regarding ACL for S3 buckets. I fixed just commenting line 123 in https://github.com/awslabs/aws-saas-boost/blob/dc6f7cbf2ceedf6af75c4160c579b11f013d60aa/resources/saas-boost.yaml

But I believe AWS released some kind of fix over weekend. 

Warm regards !

**Comment by PoeppingT** [comment, marked as answer]
Hey all, this is now fixed in https://github.com/awslabs/aws-saas-boost/commit/2cb6774ce9eb375f0cf4ae0f7b83f4434751565d

In the future, if you think you've found a bug, please open a bug report! Discussions are meant for broader discussions around SaaS Boost improvement, ideas, and more general questions.

**Comment by bricewb** [comment]
Thank you! Incredible response time!
